packages/angular/projects/clr-angular/src/emphasis/_variables.badges.scss

```diff
@@ -7,13 +7,13 @@
 $clr-badge-font-color-light: $clr-color-neutral-0 !default;
 $clr-badge-font-color-dark: $clr-color-neutral-1000 !default;
 
-$clr-badge-info-bg-color: $clr-color-action-600 !default;
+$clr-badge-info-bg-color: $clr-color-action-800 !default;
```

Comment:
These changes also need to be made in `./_properties.badges.scss`

Reply:
Done.